examples/aws/modules/aws-simple-ec2.ncl

```diff
@@ -0,0 +1,71 @@
+let Aws = import "./types.ncl" in
+let Utils = import "../utils.ncl" in
+{
+  aws.simple.ec2.instances
+    | {
+      _ : {
```

Comment:
Should you use a contract here? (remember that now dictionary type kill e.g. recursive records)

```suggestion
      _ | {
```